.editorconfig

```diff
@@ -0,0 +1,156 @@
+# EditorConfig
+# https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017
+
+###############################
+# Core EditorConfig Options   #
+###############################
+
+root = true
+
+# All files
+[*]
+indent_style = space
+
+# Code files
+[*.{cs,csx,vb,vbx}]
+indent_size = 4
```

Comment:
Let's keep this part, but remove the file endings .csx, .vb and .vbx. When reviewing it's nice that the indentation is consistent.

Reply:
Updated this PR with less rules, and changes as suggested :)